pkg/flowaggregator/flowaggregator.go

```diff
@@ -150,14 +152,29 @@ const (
 	AggregatorTransportProtocolUDP AggregatorTransportProtocol = "UDP"
 )
 
+type recordMetadata struct {
+	isValid                 bool
+	itemInActiveTimeoutPQ   *ItemToExpire
+	itemInInactiveTimeoutPQ *ItemToExpire
+}
+
 type flowAggregator struct {
 	externalFlowCollectorAddr   string
 	externalFlowCollectorProto  string
 	aggregatorTransportProtocol AggregatorTransportProtocol
 	collectingProcess           ipfix.IPFIXCollectingProcess
 	aggregationProcess          ipfix.IPFIXAggregationProcess
-	exportInterval              time.Duration
+	apMutex                     sync.Mutex
```

Comment:
maybe just spell `aggregationProcess` out, it's not that long, same comment for `epMutex` below